controllers/storagecluster/cephblockpools.go

```diff
@@ -14,18 +14,16 @@ import (
 
 type ocsCephBlockPools struct{}
 
-func generateCephReplicatedSpec(initData *ocsv1.StorageCluster) cephv1.ReplicatedSpec {
-	if arbiterEnabled(initData) {
-		return cephv1.ReplicatedSpec{
-			Size:                     4,
-			TargetSizeRatio:          .49,
-			ReplicasPerFailureDomain: 2,
-		}
-	}
-	return cephv1.ReplicatedSpec{
-		Size:            3,
-		TargetSizeRatio: .49,
+func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string) cephv1.ReplicatedSpec {
+	crs := cephv1.ReplicatedSpec{}
+
+	crs.Size = getReplicaSize(initData)
+	crs.ReplicasPerFailureDomain = uint(getReplicasPerFailureDomain(initData))
+	if "data" == poolType {
+		crs.TargetSizeRatio = .49
```

Comment:
Is the targetSizeRatio expected to be set on all data pools? I don't believe it was set previously on the object data pool. Now we need to set it on the object pool as well? That seems correct to me as well, but want to confirm it's an intentional change.

Reply:
Yes, the target size ratio was begin set for datapools in the cephfilesystem and the objectstore both.
See 
https://github.com/openshift/ocs-operator/pull/1034/files#diff-4cd9b9e8e155fb315c3d558aa21839ed627a878432c8c11e13fb811669809be7L142 
and 
https://github.com/openshift/ocs-operator/pull/1034/files#diff-fabbfe52f64e79da27f9fff14bbd71e9a6bc122fd84073a8a1a7debc4aa9ac30L38

Reply:
That commit to set the targetSizeRatio on the object store was also recent, right? I wasn't clear if that was intentional. Was there a separate BZ for that?

Reply:
I misunderstood your comment. Looked at the previous commit. It is from https://github.com/openshift/ocs-operator/pull/635 .

The BZ and the relevant comments are 
* https://bugzilla.redhat.com/show_bug.cgi?id=1850538#c24
*  https://bugzilla.redhat.com/show_bug.cgi?id=1850538#c25
*  https://bugzilla.redhat.com/show_bug.cgi?id=1850538#c30

Reply:
That's what I was looking for, thanks!